frontend/src/pages/Home.tsx

```diff
@@ -0,0 +1,188 @@
+import { Page } from "../components/Page";
+import styles from "../stylesheets/Home.module.css";
+
+export function Home() {
+  return (
+    <Page>
+      <h1 className={styles.title}>Become a Certified Interior Designer</h1>
+      <img alt="" src="/home_cover.svg" />
+      <div className={styles.textDivider}>
+        <h2 className={styles.heading}>CID Candidate Information</h2>
+      </div>
+      <ul className={styles.list}>
+        <li>
+          <p className={styles.item}>
+            Under the Certified Interior Designers Title Act law, Certified Interior Designers are
+            qualified by the California Council for Interior Design Certification (CCIDC) upon
+            evidence of a combination of interior design education, experience <strong>AND</strong>{" "}
+            passage of the IDEX® California examination.
+          </p>
+        </li>
+        <li>
+          <p className={styles.item}>
+            <strong>ONLY</strong> CCIDC can determine eligibility to be a Certified Interior
+            Designer in California. Designers who meet the criteria of the CCIDC are allowed to use
+            the “Certified Interior Designer” title and “CID” appellation and are recognized in the
+            State of California.
+          </p>
+        </li>
+        <li>
+          <p className={styles.item}>
+            Upon qualification, the Certified Interior Designer will be identified by an individual
+            number on a certificate, a stamp to identify interior design nonstructural and
+            non-seismic drawings and documents, and an identification card (ID).
+          </p>
+        </li>
+        <li>
+          <p className={styles.item}>
+            Certification in California is a Title Act, not a license or registration (Practice
+            Act). The Certified Interior Designer’s Title Act is the only thing the state
+            recognizes.
+          </p>
+        </li>
+      </ul>
+      <button className={styles.apply} type="button">
```

Comment:
This will use the button component created by the Navbar team once that PR is in.